Makefile

```diff
@@ -0,0 +1,8 @@
+help:
```

Comment:
🤔 what's the value of having a makefile? will this be inside the image? Is this mostly for the the github pipeline to have as a shortcut?